rosbag2_test_common/include/rosbag2_test_common/temporary_directory_fixture.hpp

```diff
@@ -76,7 +77,16 @@ class TemporaryDirectoryFixture : public Test
     file_options.lpszProgressTitle = nullptr;
     file_options.hNameMappings = nullptr;
 
-    SHFileOperation(&file_options);
+    const auto rc = SHFileOperation(&file_options);
+    if (0 != rc) {
+      std::cerr << "Failed to recursively delete '" << directory_path <<
+        "' Error code: " << rc << std::endl;
+    }
+
+    if (file_options.fAnyOperationsAborted) {
+      std::cerr << "Recursive delete of '" << directory_path <<
+        "' was aborted." << std::endl;
+    }
```

Comment:
does it make sense to add an `ASSERT_FALSE` in here? Not sure how valuable the preceding/succeeding tests are if this operation failed.